aeroflexSup/aeroflex_common.db

```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
     field(ONAM, "COMMS DISABLED")
 }
 
+record(bi, "$(P)CMODE_RBV") {
```

Comment:
AS per https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming#signal-qualifiers this should be just `CMODE` with optionally an alias to `CMODE:SP:RBV`, this naming convention helps setting blocks work from python as you set a block to "CMODE" and it adds `:SP` if you write from python